doc/METADATA.ini

```diff
@@ -679,6 +679,43 @@ type= enum
 	RFC2822
 description= Support of some standards for a date specification.
 
+[check/permission]
+status= idea
+usedby/plugin= path
+type= enum
+description= A check to see if a certain user has correct permissions on a file. Permissions are
+	either r (read) w (write) x (execute) and any combination of them.
+	 The respective user which must have these permissions is declared in check/permission/user.
+example = "rwx ; rw ; r ; x ; w ; wx"
+
+[check/permission/user]
+status= idea
+usedby/plugin= path
+type= string
+	empty
```

Comment:
empty or string?

Reply:
either or. If empty then the current user is taken, if a string is given it is interpreted as the desired user. It is also written in the description

Reply:
string automatically includes empty. Elektra does not have sum types, so specifying "string empty" does not make sense. (The result would be "empty")